docker/drone/src/main/java/org/arquillian/cube/docker/drone/util/VolumeUtils.java

```diff
@@ -0,0 +1,52 @@
+package org.arquillian.cube.docker.drone.util;
+
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Volume utils.
+ */
+public class VolumeUtils {
```

Comment:
I'm not a big fan of `Utils` classes. Maybe `VolumeCreator` would be a better fit here?